sdk/eventhub/event-hubs/test/public/utils/wrapper.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { isNode, TestFunctionWrapper, versionsToTest } from "@azure/test-utils";
+import { getEnvVarValue } from "./testUtils";
+
+export type SupportedTargets = "mock" | "live";
+const serviceVersions: SupportedTargets[] = ["mock", "live"];
+const testTarget = getEnvVarValue("TEST_TARGET") || "live";
+
+export function wrapper(
```

Comment:
So it's like `testWithBroker' or something? The name seems really generic.

Reply:
Yeah, I punted on this name and now you've called me out :smile:

This wrapper is meant to allow your tests to know if they should be running against the mock or the live service. What to call it...`testWithServiceType`?

Reply:
Updated the name to be less generic!